torch/_dynamo/debug_utils.py

```diff
@@ -569,7 +569,7 @@ def same_two_models(gm, opt_gm, example_inputs, only_fwd=False):
                 " Skipping this graph."
             )
         )
-        return True
+        raise
```

Comment:
??

Reply:
We need this as it moved to runtime

Reply:
OK, well, if you want to change the behavior here, you need to at least update the warning above.

But the original intent of squelching the error still seems valid; if you get stuck on an unrelated exception, you're never going to get to the actual failure that you wanted to minify. So I don't really understand your stated justification...